src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -1034,8 +1034,12 @@ static void addArticulationToChord(const Notation& notation, ChordRest* cr)
     const SymId articSym = notation.symId();
     const QString dir = notation.attribute("type");
     const QString place = notation.attribute("placement");
+    const QColor color { notation.attribute("color") };
     Articulation* na = Factory::createArticulation(cr);
     na->setSymId(articSym);
+    if (color.isValid()) {
```

Comment:
If we treat it as layout on export, we should on import too 

Here and all similar occurernces

Reply:
On second thought, I really don't believe it is a part of the _layout_. (And it ist very easy to get rid of in the exported file.)